modules/smartadserverBidAdapter.js

```diff
@@ -86,15 +86,29 @@ export const spec = {
           h: size[1]
         }));
       } else if (videoMediaType && (videoMediaType.context === 'instream' || videoMediaType.context === 'outstream')) {
+        // use IAB ORTB values if the corresponding values weren't already set by bid.params.video
+        var protocol = bid.params.video.protocol ? bid.params.video.protocol : Math.max(videoMediaType.protocol)
```

Comment:
Shouldn't it be videoMediaType.protocol**s** ?

Reply:
![image](https://user-images.githubusercontent.com/66423906/120674715-aa855200-c494-11eb-8838-6e16e722fae3.png)
Indeed

Reply:
And do you know what happen for the max function if videoMediaType.protocols is null/undefined/not an array/empty ? 
Maybe we should have a check ?

Reply:
In Javascript if:
Null will return 0
Empty will return 0
undefined will return Nan
not an array -> i don't know what it will return i need to do some checks first.